lib/core/src/Cardano/Wallet/Primitive/Migration.hs

```diff
@@ -66,6 +66,12 @@ createPlan
     -> RewardWithdrawal
     -> MigrationPlan
 createPlan constraints utxo reward = MigrationPlan
+-- To fix the migration integration test (and make migration better in Babbage)
+-- we could:
+--
+-- 1. Find the largest address in the UTxO
+-- 2. Call a `setMaxAddress :: TxConstraints -> TxConstraints` beforing passing
```

Comment:
@jonathanknowles I think you were right, this might might be simple to do something like this here